Marlin/Marlin.ino

```diff
@@ -42,6 +42,8 @@
     #include <LiquidTWI2.h>
   #elif ENABLED(DOGLCD)
     #include <U8glib.h> // library for graphics LCD by Oli Kraus (https://code.google.com/p/u8glib/)
+  #elif ENABLED(LCM1602)
+    #include <LiquidCrystal.h> 
```

Comment:
Yeah @thinkyhead the correct is actually    #elif ENABLED(LCM1602)
    #include <Wire.h>
    #include <LCD.h>
    #include <LiquidCrystal_I2C.h>

i made a mistake here